code_blocks/🔌 Integrations & Events/third-party-integrations/onesignal_2.swift

```diff
@@ -0,0 +1,32 @@
+class AppDelegate: UIResponder, UIApplicationDelegate, OSSubscriptionObserver {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+        Purchases.configure(withAPIKey: "<revenuecat_api_key>", appUserID: nil)
+
+        OneSignal.initialize("<onesignal_app_id>", withLaunchOptions: launchOptions)
+        OneSignal.add(self as OSSubscriptionObserver)
```

Comment:
I don't think this is required since `AppDelegate` conforms to `OSSubscriptionObserver`
```suggestion
        OneSignal.add(self)
```